packages/hmssdk_flutter/android/build.gradle

```diff
@@ -30,7 +30,7 @@ apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 
 android {
-    compileSdkVersion 31
+    compileSdkVersion 34
```

Comment:
- Why is the change done on SDK's build.gradle & not on the app?
- Will this cause issues on apps which have lower compileSdkVersion like 32 or 33?